ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/PDEUIMessages.java

```diff
@@ -1092,7 +1092,11 @@ public class PDEUIMessages extends NLS {
 	public static String TracingTab_AttributeLabel_TracingOptions;
 	public static String TracingTab_AttributeLabel_TracingChecked;
 	public static String TracingTab_AttributeLabel_TracingNone;
+	public static String TracingTab_TracingOptionsManager_Progress;
 
+	public static String TracingBlock_initializing_tracing_options;
+	public static String TracingBlock_initializing_tracing_template;
+	public static String TracingBlock_initializing_tracing_template_table;
```

Comment:
```suggestion
	public static String TracingBlock_initializing_tracing_options;
```
Because in all three of this cases the same method is called eventually for which the progress reporting happens I think it is sufficient to have one message here. The first one IMHO fits well. :)
```suggestion
	public static String TracingBlock_initializing_tracing_options;
```

Reply:
Done